dev.mateusz1913.Pilot.json

```diff
@@ -0,0 +1,58 @@
+{
+    "id" : "dev.mateusz1913.Pilot",
```

Comment:
`http://mateusz1913.dev/` doesn't answer

Reply:
I had some problem with DNS settings, it's fixed now